### PR TITLE
Calculate trending words with Z-Score algorithm

### LINKED ISSUE #26: Calculate trending words with Z-Score algorithm

### PR DESCRIPTION
**Pull request by 0x1b2c**
This should close #26.

### CODE REVIEW

File: bin/scheduler [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ENV['ECHIDNA_ENV'] ||= "development"
+
+require 'bundler'
+Bundler.require(:default, ENV['ECHIDNA_ENV'])
+
+require 'rufus/scheduler'
+
+scheduler = Rufus::Scheduler.start_new
+
+scheduler.cron '5 * * * *', blocking: true do
```
Comment (flyerhzm) [inline bin/scheduler:12]: why you calculate last hour's z-scores every 5 minutes?

**Comment by flyerhzm** [comment]
I merged this pull request for https://github.com/transist/echidna-spider/issues/32, but I didn't see any test code for z-scores

File: bin/scheduler (hunk 1 above)
Comment (0x1b2c) [inline bin/scheduler:12]: Oh, this is crontab file format , it not means every 5 minutes, it means the 5th minute of every hour. Check `man 5 crontab` for details.

**Comment by 0x1b2c** [comment]
I'll add test for z-scores calculation in #42.

**Pull request merged**